### ISSUE TITLE
Title: WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (a[href$="deepc\/"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [knative/ux#66]
Tags: Accessibility, WCAG 2.4.4, WCAG 4.1.2, link-name

Issue: Ensures links have discernible text 

Target application: Home - Knative - https://knative.dev/docs/

Element path: #case-studies > a[href$="deepc\/"]

Snippet: <a href="about/case-studies/deepc/"> <div class="logo"> <img src="images/case-studies/deepc.png" alt="" draggable="false"> </div> </a>

How to fix: 
Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Environment: Chrome version 97.0.4692.99
